web_src/css/base.css

```
@@ -477,6 +477,15 @@ img.ui.avatar,
   padding-bottom: 80px;
 }
 
+.status-page-error-code {
+  font-size: 3em;
+  line-height: initial;
+  text-align: center;
+  padding: 2em 0;
+  font-weight: var(--font-weight-bold);
+  color: var(--color-text-light-3);
```

```suggestion
  color: var(--color-text-light-1);
```

No reason to have so little contrast.

---

I have tried, I think `--color-text-light-3` looks better with large fonts.

Feel free to checkout and edit this PR. Maybe we could make the fonts smaller to 2em or 32px.

---

Hmm yes let me actually try it. Maybe value `2` is also good. `3` seems to subtle.

---

3 looks ok actually to preverse a bit distinction between title and detail message.